src/hooks/useAnimatedHighlightStyle.ts

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {InteractionManager} from 'react-native';
+import {Easing, interpolate, interpolateColor, useAnimatedStyle, useSharedValue, withDelay, withSequence, withTiming} from 'react-native-reanimated';
+import CONST from '@src/CONST';
+import useTheme from './useTheme';
+
+type Props = {
```

Comment:
Can you please add a comment to the individual props here so that we are compliant with this checklist item i.e. and each prop has a `/** comment above it */`?